ihmc-graphics/src/libgdx/java/us/ihmc/rdx/shader/RDXShader.java

```diff
@@ -22,12 +22,21 @@ public RDXShader(Class<?> clazz)
    }
 
    public void create()
+   {
+      create(null, null);
+   }
+
+   public void create(String[] vertexFlags, String[] fragmentFlags)
```

Comment:
Added support for adding #define flags to the shaders during runtime

Reply:
whoa